src/vis/vis.js

```diff
@@ -301,9 +305,11 @@ var Vis = View.extend({
       return this;
     }
 
-    // load modules needed for layers
+    // Load modules needed for layers
     var layers = data.layers;
 
+    // TODO - Is the module loading necessary. We are using browserify so we
+    // know dependencies are loaded.
     if (!this.checkModules(layers)) {
```

Comment:
This is loading torque.js only when there's a layer. AFAIK the motivation is to not include torque.js inside of cartodb.js and load it dynamically instead.